clients/fides-js/src/lib/gpp/stub.ts

```diff
@@ -52,10 +51,10 @@ export const makeStub = () => {
       cmpStatus: CmpStatus.STUB, // possible values: stub, loading, loaded, error
       cmpDisplayStatus: CmpDisplayStatus.HIDDEN, // possible values: hidden, visible, disabled
       signalStatus: SignalStatus.NOT_READY, // possible values: not ready, ready
-      supportedAPIs: SUPPORTED_APIS,
+      supportedAPIs: [],
       cmpId: 0, // IAB assigned CMP ID, may be 0 during stub/loading
       sectionList: [],
-      applicableSections: [],
+      applicableSections: [-1],
```

Comment:
ah this was something I saw on rereading the spec

> The value can be 0 or a Section ID specified by the Publisher / Advertiser, during stub / load. When no section is applicable, the value will be [-1].

actually re-reading it, maybe I should set it to 0 instead since this is in the stub...

Reply:
Hm, yeah that sounds right. But also sounds like we'd possibly later need to set it to `-1` if we find that no sections are applicable. Can you easily make that update?

Reply:
Yes, I'll update to `[0]`!